wiki/News_styling_criteria/en.md

```diff
@@ -132,6 +105,8 @@ jpeg-recompress -am smallfry <input> <output>
 
 Where `<input>` is the file name to be compressed and `<output>` is the compressed file name.
 
+Alternatively, [this site](https://ezgif.com/) can be used to convert & optimise images.
```

Comment:
I've been using this site for years and never had any issues with it (easy to use, no ads). I generally found it easier to use over jpeg-archive (I couldn't even set it up), I want to know if this is something you would consider adding.

Reply:
addressed in 545f0d9198fbeaae803d1214ffa2ef61cbc26ea1